deploy.php

```diff
@@ -0,0 +1,142 @@
+<?php
+namespace Deployer;
+
+require 'recipe/laravel.php';
+require 'recipe/rsync.php';
+require 'recipe/npm.php';
+
+// Project name
+set('application', 'meteo-raspi');
+
+// Configuration
+set('ssh_type', 'native');
+set('ssh_multiplexing', true);
+set('writable_mode', 'chmod');
+set('writable_chmod_mode', '0775');
+set('repository', 'https://github.com/matul91/meteo-raspi.git');
+set('keep_releases', 5);
+set('shared_files', [
+    '.env'
+]);
+set('shared_dirs', [
+    'storage',
+    'public/images/photos'
+]);
+set('writable_dirs', [
+    'bootstrap/cache',
+    'vendor',
+    'storage',
+    'public/images/photos'
+]);
+set('rsync', [
+    'exclude' => [
+        '.git',
+        'deploy.php',
+        'node_modules',
+    ],
+]);
+set('rsync_src', function () {
```

Comment:
This is default settings which should be alright
```php
set('rsync_src', __DIR__);
```
Is this piece of crap necessary? 💩

Reply:
IDK, It was added in all tutorials for deployer, so I also added it for sure